go/src/lib/lomcommon/config.go

```diff
@@ -130,6 +133,15 @@ func (p *GlobalConfig_t) GetValAny(key string) any {
     return p.anyVal[key]
 }
 
+// TODO: Goutham. Add/delete other fields
+// Proc conf file params
+type ProcConfig_t struct {
+	Name    string `json:"name"`
```

Comment:
Is this not a plugin config?

Procs config has list of such plugins' conf entries. Is that correct ?

Reply:
Yes. Its plugin. I changed the name to include plugin for readability.